src/cider/nrepl/middleware/debug.clj

```diff
@@ -448,16 +448,18 @@ this map (identified by a key), and will `dissoc` it afterwards."}
 (defn looks-step-innable?
   "Decide whether a form looks like a call to a function that we could
   instrument and step into."
-  [form]
-  (when (and (seq? form) (symbol? (first form)))
-    (let [v (resolve (first form)) ; Note: special forms resolve to nil
-          m (meta v)]
-      ;; We do not go so far as to actually try to read the code of the function
-      ;; at this point, which is at macroexpansion time.
-      (and v
-           (safe-to-debug? (:ns m))
-           (not (:macro m))
-           (not (:inline m))))))
+  ([form]
+   (looks-step-innable? nil form))
+  ([env form]
+   (when (and (seq? form) (symbol? (first form)))
+     (let [v (resolve env (first form)) ; Note: special forms resolve to nil
```

Comment:
Note that there's another `resolve` in this file - looks good to upgrade as well

Reply:
done!